### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/virtual-environments.md`

### PR DESCRIPTION
**Pull request by JCTrapero**
🌐 Add Spanish translation for docs/es/docs/virtual-environments.md

Best regards

### CODE REVIEW

File: docs/es/docs/virtual-environments.md [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+$ deactivate
+```
+
+</div>
+
+De esta manera, cuando ejecutas `python` no se ejecutará desde ese entorno virtual junto con los paquetes que contiene.
+
+## Preparado para Trabajar
+
+Ahora estás preparado para comennzar a trabajar en tu proyecto.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/virtual-environments.md:424]:
```suggestion
Ahora estás preparado para comenzar a trabajar en tu proyecto.
```

**Review by JavierSanchezCastro — CHANGES_REQUESTED** [review]
Typo

File: docs/es/docs/virtual-environments.md (hunk 1 above)
Comment (JCTrapero) [reply docs/es/docs/virtual-environments.md]: Genial! Gracias!!

**Review by JavierSanchezCastro — APPROVED** [review]

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**